src/Progress/LinearProgress.js

```diff
@@ -0,0 +1,204 @@
+// @flow weak
+
+import React, { Component, PropTypes } from 'react';
+import { createStyleSheet } from 'jss-theme-reactor';
+import classNames from 'classnames';
+
+export const styleSheet = createStyleSheet('LinearProgress', (theme) => {
+  const { palette } = theme;
+  return {
+    root: {
+      display: 'block',
+      position: 'relative',
+      width: '100%',
+      overflow: 'hidden',
+      height: 5,
+      transform: 'translate(0, 0) scale(1, 1)',
+      backgroundColor: palette.primary[100],
+    },
+    rootBuffer: {
+      backgroundColor: 'transparent',
+    },
+    bar: {
+      position: 'absolute',
+      left: 0,
+      bottom: 0,
+      top: 0,
+      height: 5,
+      transition: 'transform 0.2s linear',
+      backgroundColor: palette.primary[500],
+    },
+    dashed: {
+      '&:before': {
+        content: '""',
+        position: 'absolute',
+        marginTop: 0,
+        height: 5,
+        width: '100%',
+        background:
+          `radial-gradient(${palette.primary[100]} 0%, ${palette.primary[100]} 16%, transparent 42%)`,
+        backgroundSize: '10px 10px',
+        backgroundPosition: '0px -23px',
+        animation: 'buffer 3s infinite linear',
+      },
+    },
+    indeterminateBar1: {
+      willChange: 'left, right',
```

Comment:
I'm wondering about the use-case for this property here.
Still, I think that it would be great to add `willChange: 'width'` for the `determinate` mode on the bar1.